MathProblems/873_Length_of_Longest_Fibonacci_Subsequence.py

```
@@ -0,0 +1,22 @@
+class Solution:
+    def lenLongestFibSubseq(self, arr: list[int]) -> int:
+        hash_set=set(arr)
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)